data-migration/migrate-with-binlog-event-filter.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Filter Binlog Events
+summary: Learn how to filter binlog events when migrating data
+---
+
+# Filter Binlog Events
+
+This document describes how to filter binlog events when you use DM to perform continuous incremental data replication. For the detailed replication instruction, refer to the following documents:
+
+- [Migrate MySQL Data of Less Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-less-tb.md)
+- [Migrate MySQL Data of More Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-above-tb.md)
+- [Migrate and Merge MySQL Shards of Less Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-less-tb.md)
+- [Migrate and Merge MySQL Shards of More Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-above-tb.md)
+
+## Configuration
+
+To use binlog event filter, add a `filter` when configuring the task configuration file of DM, as shown below:
+
+```yaml
+filters:
+  rule-1:
+    schema-pattern: "test_*"
+    table-pattern: "t_*"
+    events: ["truncate table", "drop table"]
+    sql-pattern: ["^DROP\\s+PROCEDURE", "^CREATE\\s+PROCEDURE"]
+    action: Ignore
+```
+
+- `schema-pattern`/`table-pattern`: Filter matching schemas or tables
+- `events`: Filter binlog events. Supported events are listed in the table below:
+
+  | Event           | Category | Description                       |
+  | --------------- | ---- | --------------------------|
+  | all             |      | Includes all events            |
+  | all dml         |      | Includes all DML events        |
+  | all ddl         |      | Includes all DDL events        |
+  | none            |      | Includes no event          |
+  | none ddl        |      | Excludes all DDL events      |
+  | none dml        |      | Excludes all DML events      |
+  | insert          | DML  | Insert DML event      |
+  | update          | DML  | Update DML event      |
+  | delete          | DML  | Delete DML event      |
+  | create database | DDL  | Create database event |
+  | drop database   | DDL  | Drop database event   |
+  | create table    | DDL  | Create table event    |
+  | create index    | DDL  | Create index event    |
+  | drop table      | DDL  | Drop table event      |
+  | truncate table  | DDL  | Truncate table event  |
+  | rename table    | DDL  | Rename table event    |
+  | drop index      | DDL  | Drop index event      |
+  | alter table     | DDL  | Alter table event     |
+
+- `sql-pattern`：Filter specified DDL SQL statements. The matching rule supports using a regular expression.
+- `action`: `Do` or `Ignore`
+
+    - `Do`: the allow list. A binlog event is filtered if meeting either of the following two conditions:
```

Comment:
seems the `Do` section is not same as Chinese version, do you deliberately refine it?

Reply:
> seems the `Do` section is not same as Chinese version, do you deliberately refine it?

The previous Chinese is inaccurate and I have suggested Zhaoyang optimize it. The current English version is the translation of the previous Chinese. I will modify it based on the latest Chinese.